### ISSUE TITLE
[Task]: Fix accessibility notice with the pager

### ISSUE THREAD

**Issue by acouch** [HHS/simpler-grants-gov#2096]
---

Migrated from https://github.com/navapbc/simpler-grants-gov/issues/177
Originally created by @acouch on *Mon, 12 Aug 2024 17:45:06 GMT*

---
### Summary

Running lighthouse locally on the latest version of the search page turns up this:
<img width="830" alt="image" src="https://github.com/user-attachments/assets/31011556-a8d6-4571-8688-fc07a9d3eaf5">

### Acceptance criteria

- [ ] Accessibility notice fixed

**Comment by mdragon** [comment]
@acouch I think you posted a 100 Lighthouse score recently, does that mean this has been fixed?

**Comment by acouch** [comment]
This is still an issue on the search page specifically.

**Comment by emilycnava** [comment]
https://github.com/HHS/simpler-grants-gov/issues/2152

**Comment by emilycnava** [comment]
Interestingly this may be an issue with the trussworks uswds pagination component? Looking into that more, ![Image](https://github.com/user-attachments/assets/f82ec110-b572-4e95-88ad-c883ebfedfef)

**Comment by emilycnava** [comment]
After investigation, this WAS an issue with the trusswords uswds react library in version 7 (which we currently use), but has been fixed as part of the a11y omnibus commits a few months ago for versions 9.1 and higher.

This can be seen in action by going to the [uswds trussworks react storybook page](https://trussworks.github.io/react-uswds/?path=/docs/components-pagination--docs) and running lighthouse on their pagination component and it won't flag the same error, but locally in our repo it does. Code differences seen below

Local:
![Image](https://github.com/user-attachments/assets/631740b3-0a63-481e-8024-bd5c897c5704)

Hosted trussworks storybook (latest version)
![Image](https://github.com/user-attachments/assets/eabbf08c-f0b6-4657-a8c5-c0087891e932)

The crux of the error comes down to a few missing aria labels, most notably on the ellipsis list item. I did test the search page with a screen reader and it was still navigable so I do not personally feel this is a critical fix that must be done immediately, but we should probably update our react uswds library version to fix this (and other) bug fixes.

**Comment by mdragon** [comment]
@andycochran interested in your thoughts about undertaking a 7 to 9.1+ upgrade.

**Comment by andycochran** [comment]
I think we should prioritize keeping the truss dependency updated. We will get a lot for free and stay in-sync w/ the USWDS guidance we follow. Do we know what the lift might be? There are a lot of components we're not even using. Prob worth doing a spike to see what all it affects in our code.

**Comment by mdragon** [comment]
Yup, agreed, we can spin the work off from this to do that investigation and ticketing for an update.
